### ISSUE TITLE
Its possible to do this in an .ipa file?

### ISSUE THREAD

**Issue by kaykyr** [SwftAIO/TikTok-SSL-Pinning-Bypass#6]
Hello! This script is perfect. I'm running it on iOS 15.7.2 latest TikTok version.

I'd like to know, it's possible to do the same into a .ipa file?

Maybe using Hopper Disassembler and recompiling binaries?

**Comment by culturally** [comment]
Yes you can decompile the binary in Hopper/IDA Pro and then modify the bytes.

Change it to

`MOV W0, #1`
